enginetest/enginetests.go

```diff
@@ -5330,3 +5331,165 @@ func DrainIteratorIgnoreErrors(ctx *sql.Context, iter sql.RowIter) {
 		}
 	}
 }
+
+func TestSelectIntoFile(t *testing.T, harness Harness) {
+	harness.Setup(setup.MydbData, setup.MytableData, setup.EmptytableData, setup.NiltableData)
+	e := mustNewEngine(t, harness)
+	defer e.Close()
+
+	ctx := NewContext(harness)
+	err := CreateNewConnectionForServerEngine(ctx, e)
+	require.NoError(t, err, nil)
+
+	tests := []struct {
```

Comment:
This could use some tests for `ESCAPED BY`.  (Or... alternatively... we _could_ punt on `ESCAPED BY` if the customer doesn't need it, but seems like we're close enough that we should just knock it out)

Reply:
Added some more tests